composer.json

```diff
@@ -25,45 +25,45 @@
         "ext-dom": "*",
         "ext-json": "*",
         "ext-sqlite3": "*",
-        "acquia/coding-standards": "^0.7.0",
-        "composer/composer": "~2.2.5",
+        "acquia/coding-standards": "^0.8.0",
+        "composer/composer": "~2.3.5",
         "cweagans/composer-patches": "^1.7",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-        "ergebnis/composer-normalize": "^2.0.0",
-        "hassankhan/config": "^2.1",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+        "ergebnis/composer-normalize": "~2.15.0",
+        "hassankhan/config": "^3.0",
```

Comment:
I'm sure this is the source of the merge conflict ([ #235 ](https://github.com/acquia/orca/pull/235)). 🙂